CMakeLists.txt

```diff
@@ -615,4 +627,29 @@ else ()
   endif ()
 endif (IS_64BIT_BUILD EQUAL 1)
 
+
+# The prebuilt IDL headers aren't suitable for 32 bit builds, attempt to use the ones 
+# generated when building corguids by including them before the prebuilt ones
+if (WIN32 AND CLR_CMAKE_PLATFORM_ARCH_I386)
```

Comment:
Could you please share more details about the issue with prebuilt headers not working for 32 bit? We are using same set of prebuilt headers for both 32bit and 64bit in the internal builds and they work just fine.